src/components/modals/Modal.vue

```diff
@@ -57,6 +59,10 @@ export default class Modal extends Vue {
     position: relative;
     padding: 10px 22px;
     display: flex;
+    z-index: 1102;
```

Comment:
Is 1102 number really necessary here? Asking, as it's really specific, isn't 1000 enough?

Reply:
Yes, the default z-index of MUI AppBar is 1100.